common.props

```diff
@@ -4,37 +4,37 @@
 
     <MajorProductVersion>4</MajorProductVersion>
     <MinorProductVersion>1</MinorProductVersion>
-    <PatchProductVersion>1</PatchProductVersion>
-    
```

Comment:
Can you remove all the whitespace changes?

Reply:
No way! I did a reformat while touching these -- there was some of misaligned stuff and extra whitespace that was bugging me :-)

Reply:
lol..ok i thought the whitespace changes were by accident